project/accounts/migrations/0006_user_is_active.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.3 on 2022-09-07 14:28
```

Comment:
Since we have three migrations in this PR that 1) add a field 2) rename that field and 3) remove that field, lets just delete the migrations.